### PR TITLE
Add price symbol param 

### PR DESCRIPTION
**Pull request by EddWills95**
- Adds a price symbol parameter to the API call
- This will allow other currencies (a setting that could be added to the UI)

- Adds endpoint to save the currency of choice into the user file

~~Looks like prettier had a change up to. Let me know if you don't want this 😄~~

### CODE REVIEW

**Review by louneskmt — CHANGES_REQUESTED** [review]
A lot of changes seems like eslint modifications. I think you should only keep 
your "concrete" modification in this PR, without the prettier code 😉

**Comment by EddWills95** [comment]
made the changes, removing prettier. Any reason that you the project isn't following the ESLint rules?

File: routes/v1/external.js [hunk 1]
```diff
@@ -12,8 +12,11 @@ const axios = require('axios');
 const agent = new SocksProxyAgent(`socks5h://${constants.TOR_PROXY_IP}:${constants.TOR_PROXY_PORT}`);
 
 router.get('/price', auth.jwt, safeHandler(async(req, res) => {
+  // Default to USD
+  const sym = req.params.sym || "USD";
```
Comment (mayankchhabra) [inline routes/v1/external.js:16]: `sym` is nice and short, although we can probably rename it to something that's more verbal, like `currency`:

```suggestion
  const currency = req.params.currency || "USD";
```

File: routes/v1/external.js [hunk 2]
```diff
@@ -12,8 +12,11 @@ const axios = require('axios');
...
+  
   const response = await axios({
-    url: 'https://min-api.cryptocompare.com/data/price?fsym=BTC&tsyms=USD',
+    url: `https://min-api.cryptocompare.com/data/price?fsym=BTC&tsyms=${sym}`,
```
Comment (mayankchhabra) [inline routes/v1/external.js:19]: ^ Same

```suggestion
    url: `https://min-api.cryptocompare.com/data/price?fsym=BTC&tsyms=${currency}`,
```

**Review by mayankchhabra — CHANGES_REQUESTED** [review]
Looks great, @EddWills95! Thanks for the PR! Just added a tiny suggestion.

**Review by mayankchhabra — APPROVED** [review]
Awesome, thanks again @EddWills95! Congrats on your first, PR! 🎉

**Pull request merged**